packages/create-svelte/templates/default/svelte.config.js

```diff
@@ -16,7 +16,7 @@ const config = {
 };
 
 if (adapter) {
-	config.kit.adapter = require(adapter)(options);
+	config.kit.adapter = (await import(adapter)).default(options);
```

Comment:
If we need the `.default` here then do we need to do that in the adapter READMEs as well? None of them have that right now

Reply:
I believe this is only like that because it is dynamically importing the adapter. Normally you'd just do a static import, and you'd get the default export by default.

Reply:
correct